StudioCore/MsbEditor/MsbEditorScreen.cs

```diff
@@ -383,10 +383,13 @@ public override void DrawEditorMenu()
 
             if (ImGui.BeginMenu("Display"))
             {
+                /*
+                // Doesn't do anything
```

Comment:
It indeed does not do anything now but I'd clean it up later in a separate PR

Reply:
Didn't know if you wanted it for anything or not, but it was occupying menu space in the meantime. I'll strip out all grid stuff in a different PR.